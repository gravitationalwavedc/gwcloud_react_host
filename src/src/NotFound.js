import React from 'react';

function NotFound(props) {
    return (
        <p>Page Not Found</p>
    );
}

export default NotFound;