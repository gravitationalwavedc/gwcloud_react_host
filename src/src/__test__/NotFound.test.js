import React from "react";
import {expect} from "@jest/globals";
import TestRenderer from 'react-test-renderer';
import NotFound from "../NotFound";

test('Test Not Found', () => {
    const renderer = TestRenderer.create(<NotFound />);
    expect(renderer).toMatchSnapshot();
});

