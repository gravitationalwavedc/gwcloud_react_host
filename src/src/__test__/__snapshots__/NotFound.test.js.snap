// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Not Found 1`] = `
<p>
  Page Not Found
</p>
`;
